src/vs/workbench/contrib/issue/browser/issueQuickAccess.ts

```diff
@@ -18,55 +23,119 @@ export class IssueQuickAccess extends PickerQuickAccessProvider<IPickerQuickAcce
 	constructor(
 		@IMenuService private readonly menuService: IMenuService,
 		@IContextKeyService private readonly contextKeyService: IContextKeyService,
-		@ICommandService private readonly commandService: ICommandService
+		@ICommandService private readonly commandService: ICommandService,
+		@IExtensionService private readonly extensionService: IExtensionService,
+		@IConfigurationService private readonly configurationService: IConfigurationService
 	) {
 		super(IssueQuickAccess.PREFIX, { canAcceptInBackground: true });
 	}
 
 	protected override _getPicks(filter: string): Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem> | Promise<Picks<IPickerQuickAccessItem> | FastAndSlowPicks<IPickerQuickAccessItem>> | null {
+		if (this.configurationService.getValue<string>('extensions.experimental.quickAcessExtensions')) {
+			const issuePicks: Array<IPickerQuickAccessItem | IQuickPickSeparator> = [];
+			const extensionIdList: Array<string> = [];
 
-		const issuePicks: Array<IPickerQuickAccessItem | IQuickPickSeparator> = [];
+			// creates menu from contributed
+			const menu = this.menuService.createMenu(MenuId.IssueReporter, this.contextKeyService);
 
-		// creates menu from contributed
-		const menu = this.menuService.createMenu(MenuId.IssueReporter, this.contextKeyService);
+			// render menu and dispose
+			const actions = menu.getActions({ renderShortTitle: true }).flatMap(entry => entry[1]);
 
-		// render menu and dispose
-		const actions = menu.getActions({ renderShortTitle: true }).flatMap(entry => entry[1]);
-		actions.forEach(action => {
-			const pick = this._createPick(action, filter);
-			if (pick) {
-				issuePicks.push(pick);
+			// Get contributed extensions.
+			actions.forEach(action => {
+				if ('source' in action.item && action.item.source) {
+					extensionIdList.push(action.item?.source?.id);
+				}
+			});
+
+			// create picks from extensions
+			this.extensionService.extensions.forEach(extension => {
+				const pick = this._createPick(filter, undefined, extension);
```

Comment:
so I know you're grabbing the actions from the menu.... but why? If the name of the action is just gonna be the extension name, why not just not grab it from the menu and just use the `this.extensionService.extensions` to include those?